typescript-functions/main.ts

```diff
@@ -0,0 +1,68 @@
+interface Person {
+  firstName: string;
+  lastName: string;
+}
+
+function convertMinutesToSeconds(minutes: number): number {
+  const seconds: number = minutes * 60;
+  return seconds;
+}
+
+console.log('convert 60 minutes to seconds: ', convertMinutesToSeconds(60));
+
+function greet(name: string): string {
+  const greeting: string = 'Hey ' + name + '!';
+  return greeting;
+}
+
+console.log('Greeting for Tim: ', greet('Tim'));
+
+const getArea: any = (width: number, height: number): number => {
```

Comment:
Drop the `any` here. We only want to include types for the parameters and the return type. By adding `any` it's basically overriding your parameter and `return` types and not providing any type validation. Here is an example of our solution: 

```
const getArea = (width: number, height: number): number => {
  const area: number = width * height;
  return area;
};
```